src/Integration.UnitTests/Suppression/SonarQubeIssuesProviderTests.cs

```diff
@@ -141,8 +141,7 @@ public void Constructor_Connected_StartsMonitoringAndSyncs()
             SetupSolutionBinding(isConnected: true, issues: new List<SonarQubeIssue>());
 
             // Act
-            using (var issuesProvider = new SonarQubeIssuesProvider(mockSqService.Object, "keyXXX",
```

Comment:
This is a test-only change to reduce the noise in later commits.

I've modified the existing tests tests to call a `CreateTestSubject` method instead of creating the class directly.

That way, when we add a new parameter to the `SonarQubeIssuesProvider` constructor we'll only have to change the helper method.